MoltenVK/MoltenVK/GPUObjects/MVKDescriptor.h

```diff
@@ -284,25 +284,26 @@ class MVKInlineUniformBlockDescriptor : public MVKDescriptor {
 
 	void write(MVKDescriptorSet* mvkDescSet,
 			   VkDescriptorType descriptorType,
-			   uint32_t srcIndex,
+			   uint32_t dstIndex,
```

Comment:
I changed the name, because it is actually used for something different in this case (and only this one, not the other `MVKDescriptor` subclasses). I changed the caller of `write() `and `read()` as well (see below).

So basically these are two different parameters:
* byte offset in case of a `VK_DESCRIPTOR_TYPE_INLINE_UNIFORM_BLOCK_EXT`
* descriptor index for all others

They just happen to be both `uint32_t` and the third parameter. But yes, this should be documented better to explain the exception here and clear the confusion.